oms-hypervReplica-solution/README.md

```diff
@@ -0,0 +1,65 @@
+# Hyper-V Replica Solution for OMS
```

Comment:
Would "oms-hyperv-replica-solution" work as a folder name? I think it is easier to identify what the solution does rather that the abbreviated form of "oms-hvr-solution".